charts/jaeger/templates/collector-ing.yaml

```diff
@@ -15,7 +10,7 @@ metadata:
     app.kubernetes.io/component: collector
   {{- if .Values.collector.ingress.annotations }}
   annotations:
-    {{- toYaml .Values.collector.ingress.annotations | nindent 4 }}
+  {{- toYaml .Values.collector.ingress.annotations | nindent 4 }}
```

Comment:
Could you keep this indented? it reads better imho

Reply:
Of course, this is IntelliJ's style, but I can revert to your style 👍️